### DISCUSSION TITLE
Initial Window Corner

### DISCUSSION THREAD

**Discussion by hxhgxy** [ControlzEx/ControlzEx#178] [Q&A] [answered]
I try to set the cornerpreference to donotround or roundsmall, when window is loaded, it is still round.

is it a bug? we must to set it in runtime?

**Comment by batzen** [comment]
Could you attach a small repro? That's the easiest way to find out what's wrong.

**Comment by hxhgxy** [comment]
`<controlzEx:WindowChromeWindow x:Class="ControlzEx.Showcase.MainWindow"
                               xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                               xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                               xmlns:behaviors="http://schemas.microsoft.com/xaml/behaviors"
                               xmlns:controlzEx="urn:controlzex"
                               xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
                               xmlns:local="clr-namespace:ControlzEx.Showcase"
                               xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
                               xmlns:system="clr-namespace:System;assembly=mscorlib"
                               xmlns:views="clr-namespace:ControlzEx.Showcase.Views"
                               x:Name="Window"
                               Title="ControlzEx.Showcase"
                               CornerPreference="DoNotRound"
                               Width="800"
                               Height="600"
                               MinWidth="80"
                               MinHeight="60"
                               Background="{DynamicResource {x:Static SystemColors.WindowBrushKey}}"
                               FlowDirection="LeftToRight"
                               GlowColor="Navy"
                               NonActiveGlowColor="Crimson"
                               Style="{StaticResource {x:Type controlzEx:WindowChromeWindow}}"
                               WindowStartupLocation="CenterScreen"
                               WindowState="Normal"
                               SizeToContent="Manual"
                               Topmost="False"
                               mc:Ignorable="d">`


I just added CornerPreference="DoNotRound"

![ControlzEx](https://github.com/ControlzEx/ControlzEx/assets/37101628/7d0d342e-a21a-4a67-b732-5e5c56cbdbbe)

Window Corner is not what I set.

**Reply by batzen** [reply, marked as answer]
That's a bug.
It's getting tracked in #179.
Thanks for reporting this.
